frontend/src/store/users/reducer.ts

```diff
@@ -0,0 +1,30 @@
+import { createReducer } from '@reduxjs/toolkit';
+
+import { DataStatus } from 'common/enums/enums';
+import { UserByIdResponse } from 'common/types/types';
+import { getUsers } from './actions';
+
+type State = {
+  dataStatus: DataStatus;
+  users: UserByIdResponse[] | null;
+};
+
+const initialState: State = {
+  dataStatus: DataStatus.IDLE,
+  users: null,
```

Comment:
Won't it be better to have an empty array as initial value for `users`?
```suggestion
  users: [],
```